src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+
     <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        Personal information
+
+        <label class="form_field">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+           required
+          >
```

Comment:
Pay attention on indents. It should be 2 spaces. Fix all similar mistakes
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```

Reply:
done